wolox-books/src/app/components/user/user.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+  readonly ROOT_URL = 'https://wbooks-api-stage.herokuapp.com/api/v1';
+  private header = { headers: new HttpHeaders().set('Content-type', 'application/json') };
+
+  constructor(private http: HttpClient) { }
+
+  createUser(user) {
+    console.log(user);
+    this.http.post(this.ROOT_URL + "/users", user, this.header )
+      .subscribe(
```

Comment:
Keep in mind that each endpoint can be used in different parts of the app, so you shouldn't subscribe to the response in the service method